cli/types/cypress-npm-api.d.ts

```diff
@@ -0,0 +1,79 @@
+//
+// Cypress NPM api type declarations
+// https://on.cypress.io/module-api
+// https://github.com/cypress-io/cypress/issues/2141
+//
+// in the future the NPM module itself will be in TypeScript
+// but for now describe it as an ambient module
+
+declare module 'cypress' {
+  /**
+   * All options that one can pass to "cypress.run"
+   * @see https://on.cypress.io/module-api#cypress-run
+   */
+  interface CypressRunOptions {
+    browser: string
+    ciBuildId: string
+    config: object
```

Comment:
Could we also add interface for cypress `config`?

Reply:
yeah, we probably need good config interface with good jsdocs